forgettable-frontend/src/components/NavBar/NavBar.js

```diff
@@ -23,15 +24,19 @@ export default function NavBar() {
   const linkProperties = [{
     src: HomePageLogo,
     alt: 'Home Page',
+    path: '/',
   }, {
-    src: PersonsLogo,
-    alt: 'Persons Page',
+    src: PeopleLogo,
+    alt: 'People Page',
+    path: '/people',
```

Comment:
isnt the route still `/persons`?

Reply:
this is just for the page routes, so like  "localhost:3000/people". I can see how it might be confusing to have inconsistent between api and front end, but since users will see this I think its better to be grammatically correct and say "people" :)